### ISSUE TITLE
UnhandledPromiseRejectionWarning: HttpError: Resource not accessible by integration

### ISSUE THREAD

**Issue by yikeke** [shufo/auto-assign-reviewer-by-files#23]
Hi I ran this action well in my workflow yesterday, but today it runs with success but reports the following log and does not add the reviewer properly:

```
(node:2485) UnhandledPromiseRejectionWarning: HttpError: Resource not accessible by integration
    at /home/runner/work/_actions/shufo/auto-assign-reviewer-by-files/v1.0.0/dist/index.js:1:289694
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async assignReviewers (/home/runner/work/_actions/shufo/auto-assign-reviewer-by-files/v1.0.0/dist/index.js:1:21940)
(node:2485) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:2485) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

See details: https://github.com/pingcap/docs/pull/2864/checks?check_run_id=771436416

Could you please provide some guidance? @shufo

**Comment by shufo** [comment]
@yikeke 
Thank you for reporting issue!
It seems there are two problems.

1. `Resource not accessible by integration`

It occurs if someone who have no write permission of repository tries to request PR.
https://github.com/actions/first-interaction/issues/10

2. >does not add the reviewer properly:

It seems your configuration file didn't match any of the reviewers.

```yaml
'*.*':
  - TomShawn
```

e.g. `*.*` with `.github/foo.yml` => false

I just published v1.1.1 with dotfile support so you could write as this if you would like to match all files.

```yaml
'**/*':
  - TomShawn
```

```yaml
name: "Auto Assign"
on:
  - pull_request

jobs:
  assign_reviewer:
    runs-on: ubuntu-latest
    steps:
    - uses: shufo/auto-assign-reviewer-by-files@v1.1.1
      with:
        config: '.github/assign-by-files.yml'
        token: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by yikeke** [comment]
I found this issue https://github.com/actions/first-interaction/issues/10 too after I reported this issue. I found it unfortunate that Github leaves it unresolved because indeed "my use case is extremely limited" because of it. But let's hope it gets fixed soon, thanks a lot! @shufo
